online-shop/src/app/utils.ts

```diff
@@ -0,0 +1 @@
+export const url = 'http://localhost:3000';
```

Comment:
**suggestion:** you cand keep this url inside the `environment.ts` file or you could use a [proxy](https://angular.io/guide/build#proxying-to-a-backend-server)